src/components/jurisdiction/Jurisdiction.vue

```diff
@@ -67,6 +68,23 @@ export default class Jurisdiction extends Vue {
     return array
   }
 
+  /**
+   * Always exclude CA
+   * Exclude USA when states are listed in the jurisdiction list based on showUsJurisdictions flag
+   */
+  excludeJurisdictions (jurisdiction): Array<any> {
+    const excludedValues = []
+    if (jurisdiction === JurisdictionLocation.CA) {
+      excludedValues.push(JurisdictionLocation.CA)
+    }
+
+    if (this.showUsJurisdictions) {
```

Comment:
I wonder if we can rename this to be more explicit. When I first read it, it sounded to me "show us jurisdictions" as in show the reader jurisdictions 😆

Reply:
Usa?

Reply:
Yep, was thinking that too. That would be great.

Reply:
LGTM

Reply:
Thanks for fixing this.